site/sidebar.md

```diff
@@ -107,5 +107,6 @@
   - [Namespace Provisioning UI](guides/namespace-provisioning-ui.md)
   - [Variant Constructor Pattern](guides/variant-constructor-pattern.md)
   - [Value Propagation Pattern](guides/value-propagation.md)
+  - [Effective Go KRM functions](guides/effective-go-krm-function.md)
```

Comment:
Would it makes sense to have it alongside `Getting Started` instead of `guides` ?